docs/securelaunch/README.md

```diff
@@ -0,0 +1,120 @@
+**Note**: Policy file is a *JSON* file, and there is no support for comments in a JSON file.
```

Comment:
https://github.com/tailscale/hujson fyi

Reply:
or you could allow a "comment": key at any point

Reply:
Thanks. I will remove this comment. But would like to keep the json format as-is, just to avoid having one more layer of software processing...keeping things simple for now..